ci/scripts/connect-bump-versions.ts

```diff
@@ -28,35 +30,12 @@ const getGitCommitByPackageName = (packageName, maxCount = 10) =>
         `./packages/${packageName}`,
     ]);
 
-const ghWorkflowRunReleaseAction = (branch, packages, deployment) =>
-    exec('gh', [
-        'workflow',
-        'run',
-        '.github/workflows/release-connect-npm.yml',
-        '--ref',
-        branch,
-        '--field',
-        `packages=${packages}`,
-        '--field',
-        `deploymentType=${deployment}`,
-    ]);
-
-const splitByNewlines = input => input.split('\n');
+const splitByNewlines = (input: string) => input.split('\n');
 
-const findIndexByCommit = (commitArr, searchString) =>
+const findIndexByCommit = (commitArr: string[], searchString: string) =>
     commitArr.findIndex(commit => commit.includes(searchString));
 
-const initConnectRelease = async () => {
-    console.log('Using GitHub Token:', process.env.GITHUB_TOKEN ? 'Yes' : 'No');
-
-    if (process.env.GITHUB_TOKEN) {
-        // Making sure we use the proper GITHUB_TOKEN
-        exec('gh', ['auth', 'setup-git']);
-        exec('gh', ['config', 'set', '-h', 'github.com', 'oauth_token', process.env.GITHUB_TOKEN]);
-    } else {
-        throw new Error('Missing GITHUB_TOKEN');
-    }
-
+const bumpConnect = async () => {
     const checkResult = await checkPackageDependencies('connect', deploymentType);
 
     const update = checkResult.update.map(package => package.replace('@trezor/', ''));
```

Comment:
here I am seeing this in my VS code. not sure if its a big issue, just letting you know 
![image](https://github.com/trezor/trezor-suite/assets/30367552/0a157d61-8e48-4f57-836e-a220546bd186)

Reply:
Those are types issues, I am typing it a bit more to satisfy those.